Makefile

```diff
@@ -209,6 +209,15 @@ else
 	CGO_ENABLED=1 $(GOBUILD) $(RACE_FLAG) -ldflags '$(LDFLAGS) $(CHECK_FLAG)' -o '$(TARGET)' ./cmd/tidb-server
 endif
 
+.PHONY: server_with_pgo
+server_with_pgo:
+	@echo "You need to grab TiDB's cpu profile as default.pgo file from your real workload scenario.\n"
+ifeq ($(TARGET), "")
+	CGO_ENABLED=1 $(GOBUILD) -pgo=default.pgo $(RACE_FLAG) -ldflags '$(LDFLAGS) $(CHECK_FLAG)' -o bin/tidb-server ./cmd/tidb-server
+else
+	CGO_ENABLED=1 $(GOBUILD) -pgo=default.pgo $(RACE_FLAG) -ldflags '$(LDFLAGS) $(CHECK_FLAG)' -o '$(TARGET)' ./cmd/tidb-server
+endif
+
```

Comment:
IMO, there is no need to update Makefile. According to the [doc](https://go.dev/doc/pgo#building) of PGO, `go build` will detect `default.pgo` files automatically and enable PGO. Thus, we only need to put a `default.pgo` to `cmd/tidb-server` when we want to build with PGO and this can be done in CI/CD.

Reply:
doen.